calico/_includes/components/ConfigDataplaneOpenShiftManifests.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { If, Then, Else } from 'react-if';
+
+import CodeBlock from '@theme/CodeBlock';
+import Heading from '@theme/Heading';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export default function ConfigDataplaneOpenShiftManifests(props) {
+  return (
+    <>
+      <Heading
+        as='h3'
+        id='choose-dataplane'
+      >
+        Select and configure a data plane
+      </Heading>
+      <p>
+        For this installation, Calico supports two data plane options for OpenShift: <strong>eBPF</strong> and <strong>iptables</strong>. You need to choose one and configure it.
+      </p>
+      <Heading
+        as='h4'
+        id='bpf-dataplane'
+      >
+        Option1: eBPF data plane (recommended)
```

Comment:
I'm not entirely sure what's causing it, but take a look at the spacing issues for this section: 
![image](https://github.com/user-attachments/assets/b7264601-8978-4188-9e0d-1cedb7dcdc4a)

Reply:
No spaces between some of the lines?